packages/cli-plugin-metro/src/tools/loadMetroConfig.ts

```diff
@@ -73,33 +48,9 @@ export const getDefaultConfig = (ctx: ConfigLoadingContext): InputConfigT => {
           ),
         ),
       ],
-      getPolyfills: () =>
-        require(path.join(ctx.reactNativePath, 'rn-get-polyfills'))(),
-    },
-    server: {
-      port: Number(process.env.RCT_METRO_PORT) || 8081,
     },
-    symbolicator: {
-      customizeFrame: (frame) => {
-        const collapse = Boolean(
-          frame.file && INTERNAL_CALLSITES_REGEX.test(frame.file),
-        );
-        return {collapse};
-      },
-    },
-    transformer: {
-      allowOptionalDependencies: true,
-      babelTransformerPath: require.resolve(
-        'metro-react-native-babel-transformer',
-      ),
-      assetRegistryPath: 'react-native/Libraries/Image/AssetRegistry',
-      asyncRequireModulePath: require.resolve(
-        'metro-runtime/src/modules/asyncRequire',
```

Comment:
we can now remove the `metro-runtime` and `metro-react-native-babel-transformer` dependencies from this plugin's package.json